controllers/windowsmachine_controller.go

```diff
@@ -175,9 +175,19 @@ func isWindowsMachine(labels map[string]string) bool {
 	return false
 }
 
+// ignoreLabel checks if the ignore label is present or not
+func ignoreLabel(labels map[string]string) bool {
+	if value, ok := labels[IgnoreLabel]; ok {
+		if value == "true" {
+			return true
+		}
+	}
+	return false
```

Comment:
This whole function can be simplified to
```
return labels[IgnoreLabel] == "true"
```

Because of that I suggest this function either be removed, or changed to work on an object like the rest of the predicate functions, instead of a map
```
func shouldIgnore(obj client.Object){
  return ignoreLabel(obj.GetLabels()[IgnoreLabel] == true
}
```

Reply:
Makes sense.